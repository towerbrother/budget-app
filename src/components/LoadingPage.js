import React from "react";

const LoadingPage = () => {
  return (
    <div className="loader">
      <img alt="loader gif" className="loader__image" src="/loader.gif" />
    </div>
  );
};

export default LoadingPage;
