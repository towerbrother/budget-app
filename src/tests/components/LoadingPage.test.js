import React from "react";
import { shallow } from "enzyme";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import LoadingPage from "./../../components/LoadingPage";

Enzyme.configure({ adapter: new Adapter() });

test("should render LoadingPage correctly", () => {
  const wrapper = shallow(<LoadingPage />);
  expect(wrapper).toMatchSnapshot();
});
