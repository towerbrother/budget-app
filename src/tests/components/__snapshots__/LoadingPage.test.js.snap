// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `ShallowWrapper {}`;
